src/java/org/apache/cassandra/auth/AuthConfig.java

```diff
@@ -38,6 +39,15 @@ public final class AuthConfig
 
     private static boolean initialized;
 
+    /**
+     * Resets the initialized flag, enabling AuthConfig to be reconfigured multiple times within a single
+     * test case.
+     */
+    @VisibleForTesting
+    static void reset() {
```

Comment:
We have this habit of putting braces on a new line, as you probably already noticed. It would be very nice if you have same style in your work.